pkg/buildscript/merge.go

```diff
@@ -57,8 +57,8 @@ func (b *BuildScript) Merge(other *BuildScript, strategies *mono_models.MergeStr
 
 	// When merging build scripts we want to use the most recent timestamp
 	atTime := other.AtTime()
-	if atTime != nil && atTime.After(*b.AtTime()) {
-		b.SetAtTime(*atTime)
+	if atTime.After(b.AtTime()) {
```

Comment:
I think we still need the nil check. Yes any new commits and realistically 99.9% of commits have the atTime var. But commits are final; we do not edit them. Meaning old commits without this are out there, and will forever be out there.

If this is a pain to support we could maybe look into erroring out, but in that scenario we need to outline to the user how they can get out of this broken state. I don't know what that would look like exactly, so if we want to remove support for older commits we'll need to at least file a story to address those older commits with an error.